src/Microsoft.AspNetCore.Server.IntegrationTesting.IIS/IISDeploymentParameters.cs

```diff
@@ -43,6 +44,8 @@ public IISDeploymentParameters(DeploymentParameters parameters)
                 ServerConfigActionList = tempParameters.ServerConfigActionList;
                 WebConfigBasedEnvironmentVariables = tempParameters.WebConfigBasedEnvironmentVariables;
                 HandlerSettings = tempParameters.HandlerSettings;
+                GracefulShutdown = tempParameters.GracefulShutdown;
+                TestLogger = tempParameters.TestLogger;
```

Comment:
We pass logger factory into DeployAsync. Do we need to pass a logger?

Reply:
We want the logger from the TestLogger, correct?

Reply:
We want a logger it's not required to be the one with test name in a category